### PR TITLE
AWS SNS long running 

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-5929

## Description
Create a new long running integration that supports subscription confirmation and messages requests from AWS SNS.

### CODE REVIEW

File: Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  required: true
+  section: Connect
+- display: Certificate (Required for HTTPS)
+  additionalinfo: "(For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use. (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported."
+  name: certificate
+  type: 12
+  section: Connect
+  required: false
+- display: Private Key (Required for HTTPS)
+  additionalinfo: "(For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use. (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported."
```
Comment (ShirleyDenkberg) [inline Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.yml:30]: This is the same description as for Certificate.  Should this be the Private key that the service should use?

File: Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener_description.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## BaseIntegration Help
+
+Markdown file for integration configuration  help snippet. In this file add:
```
Comment (ShirleyDenkberg) [inline Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener_description.md:3]:
```suggestion
Markdown file for integration configuration help snippet. In this file add:
```

File: Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener_description.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+- Brief information about how to retrieve the API key of your product
+- Other useful information on how to configure your integration in XSOAR
```
Comment (ShirleyDenkberg) [inline Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener_description.md:6]:
```suggestion
- Brief information about how to retrieve the API key of your product.
- Other useful information on how to configure your integration in Cortex XSOAR.
```

File: Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    | Listen Port | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use. \(For Cortex XSOAR 8 and Cortex XSIAM\) If you do not enter a Listen Port, an unused port for AWS SNS Listener will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port. | True |
+    | Username | \(For Cortex XSOAR 6.x\) False \(For Cortex XSOAR 8 and Cortex XSIAM\) Optional for engines, otherwise mandatory. | False |
+    | Password |  | False |
+    | Certificate (Required for HTTPS) | \(For Cortex XSOAR 6.x\) For use with HTTPS - the certificate that the service should use. \(For Cortex XSOAR 8 and Cortex XSIAM\) Custom certificates are not supported. | False |
+    | Private Key (Required for HTTPS) | \(For Cortex XSOAR 6.x\) For use with HTTPS - the certificate that the service should use. \(For Cortex XSOAR 8 and Cortex XSIAM\) Custom certificates are not supported. | False |
```
Comment (ShirleyDenkberg) [inline Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/README.md:17]: Same comment as above

File: Packs/AWS-SNS-Listener/README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+Amazon Simple Notification Service (SNS) is a managed service that provides message delivery from publishers to subscribers. Publishers communicate asynchronously with subscribers by sending messages to a topic, which is a logical access point and communication channel. Clients can subscribe to the SNS topic and receive published messages using a supported endpoint type, such as Amazon Kinesis Data Firehose, Amazon SQS, AWS Lambda, HTTP, email, mobile push notifications, and mobile text messages (SMS).
+
+## What does this pack do
+The AWS SNS Listener supports two types of POST requests:
+* SubscriptionConfirmation: Extract the subscription URL send subscription confirmation.
+* Notification: Extract the subject and message body and creates an XSOAR / XSIAM incident.
```
Comment (ShirleyDenkberg) [inline Packs/AWS-SNS-Listener/README.md:6]:
```suggestion
* Notification: Extract the subject and message body and creates a Cortex XSOAR / Cortex XSIAM incident.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan Doc review completed.

File: Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.yml (hunk 1 above)
Comment (thefrieddan1) [reply Packs/AWS-SNS-Listener/Integrations/AWSSNSListener/AWSSNSListener.yml]: Thank you. Nice catch.

**Pull request closed without merging**